Dockerfile

```diff
@@ -8,19 +8,19 @@ COPY package.json /app
 # ---- Dependencies ----
 FROM base AS dependencies
 # install only production node packages
-RUN yarn install --production
+RUN npm install --production
```

Comment:
```suggestion
RUN npm install --only=production
```

Reply:
According to the documentation, the `--production`is ok